camel/toolkits/openbb_toolkit.py

```diff
@@ -0,0 +1,1126 @@
+            symbol=symbol, provider=provider, period=period, limit=limit
+        )
+        if return_type == "df":
+            return data.to_df()
+        else:
+            # Convert DataFrame to dict with string keys
+            if isinstance(data, pd.DataFrame):
+                return {str(k): v for k, v in data.to_dict().items()}
+            # Ensure dict has string keys
+            return (
+                {str(k): v for k, v in data.items()}
+                if isinstance(data, dict)
+                else {}
+            )
+
+    def get_income_statement(
+        self,
+        symbol: str,
+        period: Literal["annual", "quarter"] = "annual",
+        provider: str = "fmp",
+        limit: int = 5,
+        return_type: Literal["df", "raw"] = "df",
+    ) -> Union[pd.DataFrame, Dict]:
+        """Get company income statement data.
+
+        Args:
+            symbol: Company ticker symbol
+            period: Data frequency
+            provider: Data provider
+            limit: Number of periods to return
+            return_type: Return format
+
+        Returns:
+            Income statement data
+        """
+        data = self.client.equity.fundamental.income(
+            symbol=symbol, provider=provider, period=period, limit=limit
+        )
+        if return_type == "df":
+            return data.to_df()
+        else:
+            # Convert DataFrame to dict with string keys
+            if isinstance(data, pd.DataFrame):
+                return {str(k): v for k, v in data.to_dict().items()}
+            # Ensure dict has string keys
+            return (
+                {str(k): v for k, v in data.items()}
+                if isinstance(data, dict)
+                else {}
+            )
+
+    def get_cash_flow(
+        self,
+        symbol: str,
+        period: Literal["annual", "quarter"] = "annual",
+        provider: str = "fmp",
+        limit: int = 5,
+        return_type: Literal["df", "raw"] = "df",
+    ) -> Union[pd.DataFrame, Dict]:
+        """Get company cash flow statement data.
+
+        Args:
+            symbol: Company ticker symbol
+            period: Data frequency
+            provider: Data provider
+            limit: Number of periods to return
+            return_type: Return format
+
+        Returns:
+            Cash flow statement data
+        """
+        data = self.client.equity.fundamental.cash(
+            symbol=symbol, provider=provider, period=period, limit=limit
+        )
+        if return_type == "df":
+            return data.to_df()
+        else:
+            # Convert DataFrame to dict with string keys
+            if isinstance(data, pd.DataFrame):
+                return {str(k): v for k, v in data.to_dict().items()}
+            # Ensure dict has string keys
+            return (
+                {str(k): v for k, v in data.items()}
+                if isinstance(data, dict)
+                else {}
+            )
```

Comment:
could we make this as one helper function to reduce redundancy?

Reply:
Thanks @Wendong-Fan , fixed this.